components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java

```diff
@@ -704,18 +758,20 @@ public static void constructSCIMObjectFromAttributesOfLevelThree(Map.Entry<Strin
     }
 
     /**
-     * return the attribute schema for the asked attribute URI
+     * return the attribute schema for the asked attribute URI.
+     *
      * @param attributeURI
      * @param scimObjectType
      * @return
```

Comment:
Add parameter comments.

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/commit/d14dfd5420cd70c2bb864885bcb9ab336dca6ac0